### ISSUE TITLE
Special key to ignore all migrations inside an app.

### ISSUE THREAD

**Issue by timur-orudzhov** [wemake-services/django-test-migrations#48]
It would be useful to have special key for ignoring all migrations inside an app.

Consider case when we have some third party apps and want to ignore all migrations or for all third party apps at all.

Right now we should do something like this:

```python
DTM_IGNORED_MIGRATIONS = [
    ('taggit', '0002_auto_20150616_2121'),
    ('two_factor', '0002_auto_20150110_0810'),
    ('two_factor', '0003_auto_20150817_1733'),
    ('two_factor', '0004_auto_20160205_1827'),
    ('two_factor', '0005_auto_20160224_0450'),
    ('waffle', '0002_auto_20161201_0958'),
]
```

With the special key, for example, it'll get much better:

```python
THIRD_PARTY_APPS = (
    'taggit',
    'two_factor',
    'waffle',
)

# Apps specific for this project go here.
LOCAL_APPS = (
    'app_1',
    'app_2',
)

INSTALLED_APPS = THIRD_PARTY_APPS + LOCAL_APPS

DTM_IGNORED_MIGRATIONS = [
    (app, '*') for app in THIRD_PARTY_APPS
]
```

**Comment by sobolevn** [comment]
Yes! Great idea!

@timur-orudzhov are you willing to submit a fix?

**Comment by timur-orudzhov** [comment]
I'll try on next weekend.

**Comment by sobolevn** [comment]
Thanks a lot!

**Comment by sobolevn** [comment]
Related to https://github.com/wemake-services/wemake-django-template/issues/1055

We cannot add auto-names check to `wemake-python-styleguide` until this feature is resolved.

**Comment by skarzi** [comment]
Closed via #64
